modyn/trainer_server/internal/trainer/remote_downsamplers/abstract_remote_downsampling_strategy.py

```diff
@@ -22,7 +22,7 @@ def get_tensors_subset(
     # then we extract the data
     if isinstance(data, torch.Tensor):
         sub_data = data[in_batch_index]
-    elif isinstance(data, dict):
+    else:
         sub_data = {key: tensor[in_batch_index] for key, tensor in data.items()}
 
```

Comment:
data is type-hinted with a `Union` so only two possibilities.

Also pylint complains about potential uninitialized `sub_data` otherwise

Reply:
i am confused why your pylint complains about different stuff than mine, but ok :D

Reply:
also confused